src/decision/include/VisionDecision.h

```diff
@@ -23,6 +23,10 @@
 // size.
 const int NOISE_MAX = 30;
```

Comment:
Sorry if the above comment is a bit ambiguous, I've asked val to finish up #70, which should give a better example. If that's still confusing, we can go over it next weekends, params can be a bit complicated.